### ISSUE TITLE
Runtime compilation detected

### ISSUE THREAD

**Issue by cverbiest** [Riverside-Software/sonar-openedge#849]
runtime compilation works in a development environment

```run procedure.p arg1 arg2 ... argn. ```

but it fails in a runtime (production) enviroment because there are no sources available and there is no compiler license.

It would be nice to have a rule that flags this.

The rule should flag the use of ```[ argument ]...``` in the syntax below.

```
RUN 
  {   extern-proc-name
     | VALUE ( extern-expression )
     | path-name<<member-name>>
  }
  [ PERSISTENT | SINGLE-RUN | SINGLETON [ SET proc-handle]]
  [ ON [ SERVER ] {server-handle | session-handle }
         [ TRANSACTION DISTINCT ]
         [ ASYNCHRONOUS 
            [ SET async-request-handle]
            [ EVENT-PROCEDURE event-internal-procedure
                [ IN procedure-context]]
         ]
  ]
  [ ( parameter[ , parameter]... ) ]
  [ argument ]...
  [ NO-ERROR ]
```

**Comment by gquerret** [comment]
Included in develop, will be part of next release. Also added rule to detect unquoted strings in OS-COMMAND and OS-CREATE-DIR statements (as the parser consumes tokens in the same way in those statements).
